doc/hacking.rst

```diff
@@ -62,6 +62,11 @@ Here are the most common commands you'll be running:
    # run a particular cram foo.t:
    $ ./dune.exe build @foo
 
+Note that tests are currently written for version 4.14.1 of the OCaml compiler.
```

Comment:
Note that this version comes from the Makefile. So you should perhaps refer the reader to the `TEST_OCAMLVERSION` variable in the Makefile

Reply:
I added a sentence about `TEST_OCAMLVERSION`.